### DISCUSSION TITLE
G6 Start Failed.... Session Start Failed: Clock not synchronized or other error

### DISCUSSION THREAD

**Discussion by rickmooo** [NightscoutFoundation/xDrip#3124] [Bug reports]
Help. Longtime user, never any real problems before this last 24 hours. Specs: Dex6, s22 phone, Android v 13, software updated Oct. 5: TP1A.220624.014.S901USQS3CWI1

My original sensor had several days left but had several pairing requests on Oct 8 until it just stopped my sensor. Contiuned pinging all day while I was at meetings. Then at 20:51:45 I put in a new sensor, and may have replaced the transmitter to fast (without waiting full 10/15 mins). The G6 Start Failed error now starts appearing.

I did a series of phone restarts, went through the instructions here to click on/off connectivity, misc options, upgraded software and more. I restarted the sensor by removing the transmitter, waiting an hour and trying snapping in again (twice separately) after more reboots, etc. 

Same issues through today. Please help!

![Screenshot_20231009_190302_Gallery](https://github.com/NightscoutFoundation/xDrip/assets/147443461/c0e2fc52-8e5c-4358-bac2-41046b99b4b9)
![Screenshot_20231009_184756_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/0f9717bf-71da-4245-be6b-80d264b185ac)
![Screenshot_20231009_185046_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/3b9f5746-4dd1-4dd8-a193-1868ab264a79)

From logs:
![Screenshot_20231009_183516_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/d694d032-eb0d-4617-b372-f95c329fd470)
![Screenshot_20231009_184505_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/c3990b16-213b-4288-8f9f-98ee5d18ae20)
![Screenshot_20231009_184623_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/413f73f6-da79-4a17-ac8e-6c3549590bf7)
![Screenshot_20231009_184425_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/ce6373aa-faab-41ff-a2cc-ad88d15675c5)
![Screenshot_20231009_183807_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/72ba4b72-8bed-45a4-9b82-052cb3ab40ab)

**Comment by Navid200** [comment]
Can you verify that all settings match these?
https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

After doing that and a restart of the phone, does it still ping for pair frequently?
If yes, enable special pairing workaround.
Does it still ping frequently for pair?
If yes, tell me and post a screenshot of the following page.

`Settings` &#8722;> `G5/G6/Dex1 Debug Settings`

If not, you will now need to perform a restart to get the sensor going:
https://navid200.github.io/xDrip/docs/Restart-G6-sensor.html

**Reply by rickmooo** [reply]
OK I doubled checked all the settings are the same (including Trust Auto-Connect is unchecked for Samsung).

After the phone restart there was no pinging, it just connected fine. I did the Restart with plenty of time. However the same G6 Start Failed error occurred again at the first connect. Please advise, thank you.

![Screenshot_20231009_213714_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/8fcbeab7-c164-4176-9f96-e4f7b303f475)

**Reply by Navid200** [reply]
This question may sound really silly.  But, considering your description, I have to ask.
Are you absolutely sure that the transmitter you have snapped into the sensor has the transmitter ID you have entered in xDrip?

What you describe could be happening if you have accidentally placed your new transmitter somewhere like on a shelf and have mistakenly snapped in an older transmitter into the sensor.

**Reply by rickmooo** [reply]
I was a bit paranoid about that, too, so yes I checked it out and wrote the code elsewhere as backup so there was no doubt!

**Reply by Navid200** [reply]
When you performed restart following the link I provided, you removed the transmitter from the sensor and kept it out for at least 10 minutes?

If yes, it is possible the sensor is damaged.  Although, I have to say I have never experienced that myself.

**Reply by rickmooo** [reply]
I did remove it for a half hour. But considering there were problems with the first sensor (when it was force stopped) AND then this one, could it be the transmitter? Or should I check if it catches on the native app for the device?

**Reply by Navid200** [reply]
Do you have a Dexcom receiver?
Yes, you can use the Dexcom app also.

Is it possible that something is wrong with the transmitter?  Yes, it is possible.  But, I have never had a G6 transmitter fail?  I have not heard many fail.  We have had some Dexcom One starting with C fail recently.  But, G6, would be a low probability issue.

I am puzzled as much as you are.

**Reply by rickmooo** [reply]
I don't have a receiver but could use the app. I could alternatively try yet another sensor but that might mean burning a third one. 

I will try something tomorrow and keep you updated on how it turns out (of course I'm traveling right now, so my days are full of meetings).

**Reply by Navid200** [reply]
When you started the sensor after removing the transmitter and waiting and putting it back in, was there proper connectivity?
https://navid200.github.io/xDrip/docs/Proper-connectivity.html

**Reply by rickmooo** [reply]
Yes, that was all set. Here's an image just prior:
![Screenshot_20231009_205651_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/147443461/69cac09a-2531-4c21-b0dd-e95f7f3b1d78)

**Reply by rickmooo** [reply]
Thanks for your help earlier. I tried on the native app and it said transmitter failed. So getting it replaced.
